clickhouse_mysql.etc/clickhouse-mysql.conf

```diff
@@ -1,84 +1,28 @@
-# INTRODUCTION
-#
-# 1. Parameters accept both variants for naming - with '_' as a words separator or '-'.
-# The former is used for ENV VARS, the latter - for CLI options.
-# Config file accepts both, thus 'log_file' equals to 'log-file'
-#
-# 2. Lists are specified with ',' as a separator
-# For example, in case we'd like to specify two entries for `src_tables_prefixes` option:
-# src_tables_prefixes=performance_log_, indexing_log_
-#
-# 3. Boolean parameters accept case-insensitive 'yes', 'no', 'on', 'off', 1 or 0 values
-#
-# 4. Each parameter must have a value, thus
-#    'dry=yes' is ok,
-#    'dry=' is not that good, but will be interpreted as an empty line, while
-#    'dry' is a syntax error - not recognized to as a value - no '=' and right part value specified
-#
-# 5. Comments are '#'-only. So, php-like ';' is not accepted as a comment symbol in config file
-#
-# general app section
-#
+tb_host=https://api.tinybird.co
+tb_token=
 
-#config_file=clickhouse-mysql.conf
-#log_file=/var/log/clickhouse-mysql/main.log
+log_file=/mnt/disks/tb/tinybird_mysql_connector/main.log
 log_level=debug
 nice_pause=1
-#dry=yes
-#daemon=yes
-#pid_file=/tmp/clickhouse-client.pid
-#binlog_position_file=/tmp/clickhouse-mysql-binlog.pos
+
+binlog_position_file=/mnt/disks/tb/tinybird_mysql_connector/syncer_files/bl-raw_landing-local-pos
+
 mempool=yes
-#mempool_max_events_num=10000
-#mempool_max_rows_num=
-#mempool_max_flush_interval=60
-#csvpool=yes
-#csvpool_file_path_prefix=qwe_
+mempool_max_events_num=10
```

Comment:
With this, I'd say we are changing the default values right? Is this done on purpose?

Reply:
changed to avoid possible problems